ui/lib/open-api-explorer/addon/templates/components/swagger-ui.hbs

```diff
@@ -11,7 +11,8 @@
     <div class="field is-marginless">
       <p class="control has-icons-left">
         <input
-          oninput={{queue (action "updateFilter") (action "proxyEvent")}}
+          oninput={{action "proxyEvent"}}
+          onchange={{action "updateFilter"}}
```

Comment:
should we use the newer syntax?

```suggestion
    {{on "change" (action "updateFilter")}}
```

Reply:
we can, I was trying to change this file as little as possible, since it will need a glimmer refresh in the future. This is also the basic `<input>` rather than the ember's built-in [<Input>](https://guides.emberjs.com/release/components/built-in-components/) so I'll go ahead and see if that will upgrade smoothly too.

Reply:
Interesting, I was able to update the action to the newer syntax, but the input broke in the same way as before when I tried to update it to the [builtin](https://guides.emberjs.com/v3.28.0/components/built-in-components/) component 🤔 Shipping without that update for now